### ISSUE TITLE
image file should be named after configuration rather than scheduler ID

### ISSUE THREAD

**Issue by Popolechien** [kiwix/cardshop#243]
Scheduler ID is unique, but the odds of remembering what's on an image based on that seemingly random number are quite low. Configuration names, on the other hand, are handy and familiar.

I could see the issue of users using the same config name and near-simultaneously asking for a card creation, but in that case a discreet `confignamex` (with `x`= queue ID or something similar) could do the trick.

**Comment by Popolechien** [comment]
@kelson42 I do not think it solves the issue of trying to figure out what is in the config. To give a concrete example, Orange ordered three configs (`Ecoles numeriques EN`, `Ecoles numeriques FR`, and a third one for deployments in France). They then end up with three images, but all three need to be flashed on a different number of SD card. They could of course rename each up download, but having a `config_name_shortid` would be equally simple (or even simpler).

**Comment by rgaudin** [comment]
That's what @kelson42 proposes : use something similar as youzim, which is `{domain}_{shortId}`. Obviously here the domain would be the config name.

**Comment by Popolechien** [comment]
ah ok then yes, perfect.
